spec/spec_helper.rb

```diff
@@ -30,7 +32,7 @@
   end
   config.include FactoryBot::Syntax::Methods
 
-  config.before :each do
```

Comment:
Is this okay?

Reply:
https://rspec.rubystyle.guide/#redundant-beforeeach